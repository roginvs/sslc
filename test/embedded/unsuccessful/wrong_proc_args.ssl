/**
 * @arg {integer} a
 * @arg {integer} b
 */
procedure sum(variable a, variable b) begin
  variable out;
  out = a + b;
  return out;
end

procedure start begin
  call sum(1);

end

