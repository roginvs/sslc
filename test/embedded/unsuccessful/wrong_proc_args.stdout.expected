Set include directory: ../include
Compiling wrong_proc_args.ssl
[Error] <Semantic> <wrong_proc_args.ssl>:12:14: Wrong number of arguments to procedure sum, must be from 2 to 2.

*** THERE WERE ERRORS (1 of them) ***
